### ISSUE TITLE
support combining multiple result files into a single one a'la coverage

### ISSUE THREAD

**Issue by zsoldosp** [PaesslerAG/django-performance-testing#22]
it was out of scope for #21 which already turned out to be quite big

the suggestion is to have a `djpt_combine_result_files` command that takes filepaths as `*args` (`LabelCommand`) and reads all of them to memory, then write it out into a single file named `django_performance_testing.serializer.get_datafile_path`

The idea behind using labels instead of accepting file patterns and then converting them to filepaths based on `glob` or `re` or similar is to keep things simple here. 

people running things in parallel would be expected to have different file names for each run in their settings (e.g.: `djpt.results_collected.$$` to get the running process's id), and then do the bash magic themselves (e.g.: `manage.py $(ls djpt.results_collected*)`). Probably this should be documented :)
